From ad3af318e15496dbf554b40ba3661865e66dd6fe Mon Sep 17 00:00:00 2001
From: Loc Vu <loc.vu.zn@renesas.com>
Date: Thu, 16 Jul 2020 13:10:21 +0700
Subject: [PATCH 086/504] ARM: dts: iwg21m: Add sdhi2_cd to pin groups

This patch adds card-detection pin to pfc of sdhi2.

Signed-off-by: Loc Vu <loc.vu.zn@renesas.com>
---
 arch/arm/boot/dts/r8a7742-iwg21m.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/r8a7742-iwg21m.dts b/arch/arm/boot/dts/r8a7742-iwg21m.dts
index 28ac7e8..e1a3a8e 100644
--- a/arch/arm/boot/dts/r8a7742-iwg21m.dts
+++ b/arch/arm/boot/dts/r8a7742-iwg21m.dts
@@ -84,13 +84,13 @@
 	};
 
 	sdhi2_pins: sd2 {
-		groups = "sdhi2_data4", "sdhi2_ctrl";
+		groups = "sdhi2_data4", "sdhi2_ctrl", "sdhi2_cd";
 		function = "sdhi2";
 		power-source = <3300>;
 	};
 
 	sdhi2_pins_uhs: sd2_uhs {
-		groups = "sdhi2_data4", "sdhi2_ctrl";
+		groups = "sdhi2_data4", "sdhi2_ctrl", "sdhi2_cd";
 		function = "sdhi2";
 		power-source = <1800>;
 	};
-- 
2.7.4

